python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -359,24 +360,37 @@ class RandomForestClassifier(Base):
             raise ValueError("Wrong value passed in for max_features"
                              " please read the documentation")
 
-    def _get_model_info(self):
+    def _convert_to_treelite(self, task_category):
         cdef ModelHandle cuml_model_ptr = NULL
-        task_category = 1
         cdef RandomForestMetaData[float, int] *rf_forest = \
             <RandomForestMetaData[float, int]*><size_t> self.rf_forest
         build_treelite_forest(& cuml_model_ptr,
                               rf_forest,
                               <int> self.n_cols,
                               <int> task_category,
                               <vector[unsigned char] &> self.model_pbuf_bytes)
-
         mod_ptr = <size_t> cuml_model_ptr
-        fit_mod_ptr = ctypes.c_void_p(mod_ptr).value
+        treelite_handle = ctypes.c_void_p(mod_ptr).value
+        return treelite_handle
+
+    def _get_model_info(self):
```

Comment:
Not sure if `_get_model_info` is a good name for a method returning protobuf bytes, maybe `_get_protobuf_bytes` would be better. However, this refers to a pre-existing code, and I'm not sure if it is best to fix in this pull request.

Reply:
I agree with Andy. _get_protofbuf_bytes would be better.